docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -104,7 +104,7 @@ <h2>Animating the cube</h2>
 
 		<p>If you insert all the code above into the file you created before we began, you should see a green box. Let's make it all a little more interesting by rotating it.</p>
 
-		<p>Add the following right above the `renderer.render` call in your `animate` function:</p>
+		<p>Add the following code right above the `renderer.render` call in your `animate` function:</p>
```

Comment:
Thinking about this change again. I'm not sure the addition is required. Maybe it's best if an English native speaker reviews the PR.

Reply:
Not required, but I'm good with the change -- a bit clearer probably. 👍